_examples/spreadsheet/cells-with-empty/main.go

```diff
@@ -0,0 +1,24 @@
+// Copyright 2017 FoxyUtils ehf. All rights reserved.
+package main
+// This example demonstrates flattening all formulas from an input Excel file and outputs the flattened values to a new xlsx.
```

Comment:
This comment is not applicable.

Should be something like
// This example demonstrates outputing all cells in a excel spreadsheet, including empty cell.